internal/restorer/restorer.go

```diff
@@ -336,6 +336,15 @@ func (res *Restorer) RestoreTo(ctx context.Context, dst string) error {
 	return err
 }
 
+func (res *Restorer) addFile(node *restic.Node, size uint64) {
```

Comment:
Couldn't we just pretend the ADS are also regular files?

Reply:
Usually, users try to verify backups by relying on the file count as a metric. If ADS is treated as a regular file, the counts would be added and it could cause confusion.
Users would have to use dir /R to list the ADS as well and get the count. When you try to get the count by using Right Click -> Properties, which is what most users will do, you will not see the ADS count.
Similarly, if you restore on non-windows systems, ADS files will not appear, and the counts would differ.

Besides by definition I feel Alternate Data Streams are attachments to the main files and not equal in status to regular files.